extensions/amp-analytics/0.1/cookie-writer.js

```diff
@@ -224,13 +235,36 @@ export class CookieWriter {
         // provide a way to overwrite or erase existing cookie
         if (value) {
           const expireDate = Date.now() + cookieExpireDateMs;
+          // SameSite=None must be secure as per
+          // https://web.dev/samesite-cookies-explained/#samesitenone-must-be-secure
+          const secure = sameSite === SameSite.NONE;
```

Comment:
Non blocking here, I believe we could also expose `secure` setting if requested later. 
If so, the secure is set to `true` only when it is undefined.

Reply:
Exposing `secure` setting will require a slightly larger refactor, so I'd rather leave it out of scope of this fix 🙂